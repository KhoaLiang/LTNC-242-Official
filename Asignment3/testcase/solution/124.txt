Undeclared: ASSIGN xMe_3 34.4