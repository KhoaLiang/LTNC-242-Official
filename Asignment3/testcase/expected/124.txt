InvalidInstruction: ASSIGN xMe_3 34.4